common/djangoapps/microsite_configuration/backends/database.py

```diff
@@ -18,6 +18,9 @@ def has_configuration_set(self):
         """
         Returns whether there is any Microsite configuration settings
         """
+
+        # CDODGE: I believe this will be called on every request into edx-platform
+        # so it seems more expensive than it should be. Like set_config_by_domain
         if Microsite.objects.count():
```

Comment:
what about doing `Microsite.objects.all()[:1].exists():`
This creates a `SELECT (1) AS a FROM microsite_configuration_microsite LIMIT 1`

This way is not a full table count each time